pyanaconda/installclasses/scientific.py

```diff
@@ -0,0 +1,59 @@
+#
+# scientific.py
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from pyanaconda.installclass import BaseInstallClass
+from pyanaconda.product import productName
+from pyanaconda import network
+from pyanaconda import nm
+
+class RHELBaseInstallClass(BaseInstallClass):
```

Comment:
Whoops, silly typo.  I'll fix it in a new commit.

There may be some shift over time between SL and RHEL.  In particular installUpdates is on for SL.